main.go

```diff
@@ -52,7 +52,7 @@ func run(c *cli.Context) error {
 	}
 
 	sleep := os.Getenv("CATTLE_SCHEDULER_SLEEPTIME")
-	time := 1
+	time := 2
```

Comment:
we switched to new locking and temporary-reservation logic to prevent reserving things twice. Right now this time doesn't really matter, bc it indicates the expire time we rollback temporary-reservation if *scheduler.reverve* doesn't come and rollback it, which i think 2s is the safest time.